src/lev-dashboard/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import LevPage from '../lev-page';
+import DashboardContent from "./DashboardContent";
+
+const LevDashboard = (props) =>
+<LevPage
+  title="LEV Dashboard"
+  topnav={{
+    department: 'HMPO',
+    serviceTitleText: 'Life Event Verification - Dashboard',
+    signOutLink: 'oauth/logout'
+  }}
+>
+  <script dangerouslySetInnerHTML={{ __html: `window.dashboardProps=${JSON.stringify(props)};` }}></script>
+  <script src="/public/js/bundle.js"></script>
```

Comment:
It feels like this line should be a part of the renderer.

Perhaps it could take an argument to the path to the bundle?

Reply:
What happens when we have multiple UIs?
Will they all use the same JS? Or, will it be useful to be able to specify (arbitrary/many) different JS resources?

Reply:
I don't follow.

The bundle will be for the app as a whole. So lev-web will have one and lev-report will have one.

Reply:
Let's keep this for the moment, in the hope that a unified render and hydrate is soon to replace it.